src/main/java/com/exadel/discount/entity/Discount.java

```diff
@@ -0,0 +1,46 @@
+package com.exadel.discount.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+import java.util.UUID;
+
+@Entity
+@Table(name = "discounts")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(exclude = {"tags"})
```

Comment:
If you exclude it from equals and hash code, probably you need to exclude it from toString as well

Reply:
+